lib/core/AccountManager.js

```diff
@@ -60,9 +62,12 @@ exports.AccountManager = class AccountManager {
   }
 
   showUsers (context) {
+    const userQuery = (context.user && context.user.email ) || "";
```

Comment:
What is this line for? What is the desired behavior for `userQuery`?

Reply:
It will safely access `context.user.email`. accessing `context.user.email` will throw an error if `context.user` is `undefined`

Reply:
I think it won't throw any error because you set `|| ""`?
What I understand is, if we don't pass the `context.user`, the `showUsers` will fetch any active users?

Reply:
Yups, the `|| ""` shouldn't be required